app/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
         transitive = true;
     }
     //compile 'com.android.support:customtabs:25.3.1'
-    compile 'org.openobservatory.measurement_kit:android-libs:0.7.0-1'
+    compile 'org.openobservatory.measurement_kit:android-libs:0.7.2-1'
     compile 'com.android.support:appcompat-v7:25.3.1'
     compile 'com.android.support:recyclerview-v7:25.3.1'
     compile 'uk.co.chrisjenx:calligraphy:2.2.0'
```

Comment:
@lorenzoPrimi: can you also make sure we're not using any alpha or beta software in this file, because having alpha or beta in here breaks the `f-droid` build. Here it seems you only need to bump `constraint-layout`.